### ISSUE TITLE
Please implement %m (numeric month), patch included

### ISSUE THREAD

**Issue by hukarere** [thaterikperson/elm-strftime#3]
Hello, please implement %m, as per strftime(3):
  %m     The month as a decimal number (range 01 to 12).

My patch is attached.

```
--- elm-stuff/packages/thaterikperson/elm-strftime/1.0.3/src/Strftime.elm	2017-01-31 20:51:54.000000000 +0300
+++ StrftimeGrg.elm	2017-02-26 20:32:12.591857312 +0300
@@ -1,4 +1,6 @@
-module Strftime exposing (format)
+-- grg: thaterikperson/elm-strftime with %m added
+
+module StrftimeGrg exposing (format)
 
 {-| An (incomplete) implementation of the strftime format based on rules
 from http://strftime.org.
@@ -31,6 +33,7 @@
         |> Regex.replace All (regex "%w") (\_ -> numberWeekday date)
         |> Regex.replace All (regex "%-d") (\_ -> toString <| Date.day date)
         |> Regex.replace All (regex "%d") (\_ -> zeroPad <| Date.day date)
+        |> Regex.replace All (regex "%m") (\_ -> zeroPad <| numericMonth date)
         |> Regex.replace All (regex "%y") (\_ -> String.right 2 <| toString <| Date.year date)
         |> Regex.replace All (regex "%Y") (\_ -> toString <| Date.year date)
         |> Regex.replace All (regex "%-H") (\_ -> toString <| Date.hour date)
@@ -209,6 +212,44 @@
         Dec ->
             "December"
 
+numericMonth : Date -> Int
+numericMonth date =
+    case Date.month date of
+        Jan ->
+            1
+
+        Feb ->
+            2
+
+        Mar ->
+            3
+
+        Apr ->
+            4
+
+        May ->
+            5
+
+        Jun ->
+            6
+
+        Jul ->
+            7
+
+        Aug ->
+            8
+
+        Sep ->
+            9
+
+        Oct ->
+            10
+
+        Nov ->
+            11
+
+        Dec ->
+            12
 
 abbreviatedMonth : Date -> String
 abbreviatedMonth date =
```

**Comment by hukarere** [comment]
First attempt to format the patch was bad, now corrected.

**Comment by thaterikperson** [comment]
Thanks for the patch. I published version 1.0.4 that has what you're looking for: http://package.elm-lang.org/packages/thaterikperson/elm-strftime/1.0.4.

**Comment by hukarere** [comment]
Cool, thank you!
